app/assets/stylesheets/admin/logs/_logs.scss

```diff
@@ -57,27 +68,40 @@
     margin: 1em;
   }
 
-  .log-info {
-    padding: 0;
-    margin: 1em 0;
-  }
-
-  p {
-    margin: 0;
-    padding: 0;
-  }
-
   .actions-box {
     text-align: left;
     margin: 0 0.5em;
   }
-}
 
-.log.expandable {
   .summary {
     margin: 0 .5em 0 0;
   }
 
+  header {
+    color: #b3b3b3;
+    font-size: 88%;
```

Comment:
Can we use a standard size? Bootstrap has a bunch. Don't want random different font sizes floating around.